lib/proxy/log_buffer/buffer.rb

```diff
@@ -86,10 +86,22 @@ def failed_module(a_module, message)
       @failed_modules[a_module] = message
     end
 
+    def size
```

Comment:
Also remove the attr_readers from the class, else the methods will be defined twice.